src/marine/avhrr_radiance2ioda.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2019-2021 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+from __future__ import print_function
```

Comment:
This isn't required anymore. It was originally put in to be able to use python3 print syntax while running python2.

Reply:
Ah I see, Thanks for the information. I'll delete it